packages/ui/src/components/atoms/Avatar/index.tsx

```diff
@@ -1,30 +1,40 @@
 import { FC, HTMLAttributes } from 'react'
 import styled from 'styled-components'
-import { colors, device } from '../../../styles'
+import { dimensions, colors, device } from '../../../styles'
 import defaultAvatar from './default.svg'
 
 export type TAvatar = HTMLAttributes<HTMLDivElement> & {
+  onClick: () => void
   src?: string
   alt: string
+  forwardedRef?: React.RefObject<HTMLImageElement>
+  width: number
+  height: number
+  borderRadius: number | string
 }
 
-const StyledAvatar = styled.img<TAvatar>`
-  width: 97px;
-  height: 97px;
-  border-radius: 50%;
+const StyledAvatar = styled.img`
+  padding: 1px;
+  height: ${(props) => `${props.height}px`};
+  width: ${(props) => `${props.width}px`};
+  border: 1px solid ${colors.gray.gray4};
+  background-color: ${colors.white};
+  cursor: pointer;
   object-fit: cover;
   appearance: auto;
+  transition: transform 0.3s ease;
   text-align: center;
-  background-color: ${colors.gray.gray5};
-  border: 1px solid ${colors.gray.gray4};
+
+  &:hover {
+    transform: scale(1.03);
+  }
 
   @media only ${device.Tablet} {
-    width: 118px;
-    height: 118px;
-    border: none;
+    border-radius: ${dimensions.borderRadius.base};
+    width: 48px;
   }
 `
-
-export const Avatar: FC<TAvatar> = ({ src = defaultAvatar, alt }) => (
-  <StyledAvatar src={src} alt={alt} />
+export const Avatar: FC<TAvatar> = ({ src = defaultAvatar, alt, onClick, forwardedRef, width = 50, height = 50, borderRadius = 50 }) => (
+  <StyledAvatar src={src} alt={alt} onClick={onClick} ref={forwardedRef} width={width} height={height}
+    style={dimensions ? { borderRadius: `${borderRadius}px` } : {}} />
 )
```

Comment:
Please, check if you have Prettier!

Reply:
yes , I do Have it, don't know why is not working sometimes.